README.md

```diff
@@ -1,76 +1,90 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# My Library.
 
-> One paragraph statement about the project.
+> A project where you can create and destroy cards about books.
 
-![screenshot](./app_screenshot.png)
+<br/>
 
-Additional description about the project and its features.
+This project is part of the Microverse curriculum "Javascript." It is made with learning proposes. To practice the creation of objects with constructors and manage basic data from the DOM.
+<br/>
+Here the user can create cards about books with a simple form. Adding Title, the number of pages, and status of reading.
+<br/>
+It is also possible to delete the created cards. The project was created using HTML, CSS, Javascript, and the API, LocalStorage.
 
-## Built With
+<p align="center">
+  <img src="app_screenshot.png" width="800">
+</p>
 
-- Major languages
-- Frameworks
-- Technologies used
+### Prerequisites
 
-## Live Demo
+- Access to a Terminal. (Basic knowledge on the usage)
+- GitHub basic knowledge. (Preferable)
+- Visual Studio or another text editor. (Preferable)
 
-[Live Demo Link](https://livedemo.com)
+## Download this project!
 
+- At first, you have to clone this repository. Open your terminal and paste this without the $ sign. 
 
-## Getting Started
+      $ git clone https://github.com/AlejandroNo4/library-js.git
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+- Now that you have the repository in your local machine. Change the direction of the local file with your terminal. Just type "$ cd" (without the $ sign) and the path to reach the library-js folder.<br/>
+Something like:
 
+      $ cd User/Documents/ibrary-js
 
-To get a local copy up and running follow these simple example steps.
+- You are done! Now to run the project, just type:
 
-### Prerequisites
+      $ open index.HTML
 
```

Comment:
This instruction about typing **open index.HTML** did not work for me. I would suggest you add only the instruction on running the command **code .** to open it on vs code then right-click on index.html file and choose the **open with live server** option.